WooCommerce/Classes/ViewRelated/Orders/Order Details/Shipping Labels/WooShipping Create Shipping Labels/WooShipping Customs/WooShippingCustomsFormViewModel.swift

```diff
@@ -99,15 +99,21 @@ private extension WooShippingCustomsFormViewModel {
     func listenForInternationalTransactionNumberIsRequired() {
          $itemsViewModels
             .map { childViewModels in
-                childViewModels.map { $0.$internationalTransactionNumberIsRequired.eraseToAnyPublisher() }
+                childViewModels.map { $0.$hsTariffNumberTotalValue.eraseToAnyPublisher() }
             }
             .flatMap { childPublishers in
                 childPublishers.combineLatest()
             }
-            .map { childValidityArray in
-                childValidityArray.contains { $0 }
-            }.sink { [weak self] value in
-                self?.internationalTransactionNumberIsRequired = value
+            .sink { [weak self] value in
+                // Remove nils
+                let compactedValues = value.compactMap { $0 }
```

Comment:
tip: You can use `compacted()` when you want to remove nils; it does the same as `compactMap { $0 }`.

Reply:
Good point thanks! Done in 46288e7fa6